### PR TITLE
Mixins proof of concept

### PR DESCRIPTION
**Pull request by jakobw**
⚠️ Should be merged after #486

### CODE REVIEW

File: docs/src/mixins/01-typography.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+import { Meta, Canvas } from '@storybook/addon-docs/blocks';
+import './01-typography.scss';
+
+<Meta title="Guidelines/Typography" />
+
+## Typography
+
+You can use <a href="https://github.com/wmde/wikit/blob/master/vue-components/src/styles/mixins/Typography.scss">typography mixins</a> to apply WiKit text styles in context and customize any of their available attributes.
```
Comment (sai-san) [inline docs/src/mixins/01-typography.stories.mdx:8]: Maybe, but more technical: "You can include the following mixins in your local CSS to create typography classes"

**Comment by sai-san** [comment]
UI-review got stuck in this PR for some reason. Probably me making a mistake while trying to solve conflicts 😞

**Review by micgro42 — COMMENTED** [review]
I like it overall, just one tiny note.

For future considerations, maybe codex work, I would find it helpful to have some indication when to use what, especially in a section titled "Guidelines". That is, it feels very non-obvious to me when to use `body-s` and when to better use `small-text`. That being said, I'm not a designer, and maybe it is crystal clear to someone already skilled and talented in the field^^
Though this is not needed as part of this pull request.

File: docs/src/mixins/01-typography.stories.mdx (hunk 1 above)
Comment (sai-san) [reply docs/src/mixins/01-typography.stories.mdx:8]: I think "any" is correct, but the whole sentence makes it sound confusing. I'll edit. Thanks, Michael!

Comment (sai-san) [inline docs/src/mixins/01-typography.stories.mdx:8]:
```suggestion
You can use <a href="https://github.com/wmde/wikit/blob/master/vue-components/src/styles/mixins/Typography.scss">typography mixins</a> to apply WiKit text styles in context.
```

Comment (micgro42) [reply docs/src/mixins/01-typography.stories.mdx:8]: Sounds good to me, thanks!

**Review by micgro42 — APPROVED** [review]

**Comment by sai-san** [comment]
> I like it overall, just one tiny note.
> 
> For future considerations, maybe codex work, I would find it helpful to have some indication when to use what, especially in a section titled "Guidelines". That is, it feels very non-obvious to me when to use `body-s` and when to better use `small-text`. That being said, I'm not a designer, and maybe it is crystal clear to someone already skilled and talented in the field^^ Though this is not needed as part of this pull request.

This makes total sense, Michael! I just didn't have the mental space to realize that, so I appreciate the heads-up. I'll add all that information in a separate PR.

**Pull request merged**